packages/block-library/src/image/block.json

```diff
@@ -33,6 +33,13 @@
 			"selector": "figcaption",
 			"__experimentalRole": "content"
 		},
+		"lightbox": {
+			"type": "object",
+			"enabled": {
+				"type": "boolean",
+				"default": false
```

Comment:
Let's remove this line. Right now, this line is useless as the default value should be on the first level of the attribute definition. That said, we shouldn't move it to the parent definition in this case because it will change the behavior (won't inherit the global value by default but will always disable it by default), so for now, let's just drop the default property from here.

Reply:
You're right. Good catch, thanks!